tests/test_corona.py

```diff
@@ -404,7 +404,7 @@ def test_population():
 
     new_jersey = c.population("US", "New Jersey")
     assert isinstance(new_jersey, int)
-    assert new_jersey > 17000000
+    assert new_jersey > 8000000
```

Comment:
Shouldn't we also have a test that the population is not too big? That would have caught the bug under discussion here, and will catch the same problem should be the bug be re-introduced by accident. I.e. something like 

`assert new_jersey < 10000000`

Reply:
You're right. Done!